plugin/manager.go

```diff
@@ -58,7 +58,7 @@ func Init(root string, remote libcontainerd.Remote, rs registry.Service, liveRes
 	root = filepath.Join(root, "plugins")
 	manager = &Manager{
 		libRoot:           root,
-		runRoot:           "/run/docker",
+		runRoot:           "/run/docker/plugins",
```

Comment:
@anusha-ragunathan @vieux I changed this to accommodate legacy plugins that have hardcoded `/run/docker/plugins` already. I had initially chosen `/run/docker` because it was shorter.